src/main/java/com/dicemc/corruptedlands/Registration.java

```diff
@@ -148,4 +155,17 @@ public static void doDispense(World worldIn, ItemStack stack, int speed, Directi
 			protected void playDispenseSound(IBlockSource source) {source.getWorld().playEvent(1000, source.getBlockPos(), 0);}
 			protected void spawnDispenseParticles(IBlockSource source, Direction facingIn) {source.getWorld().playEvent(2000, source.getBlockPos(), facingIn.getIndex());}
 	    }
+	    
```

Comment:
As a matter of preference, move these above the dispenser class and the mapBlockPairs method